src/Tizen.Applications.Common/Tizen.Applications/DirectoryInfo.cs

```diff
@@ -180,5 +183,73 @@ public string ExpansionPackageResource
                 return _expansionPackageResourcePath;
             }
         }
+
+        /// <summary>
+        /// Gets the absolute path to the application's resource control directory, which is used to share the resources of the allowed resource packages that available.
+        /// </summary>
+        /// <param name="res_type">The resource package's type</param>
+        /// <returns> The absolute path to the application's resource control directory, which is used to share the resources of the allowed resource packages that available.</returns>
+        /// <exception cref="ArgumentException">Thrown in case of an invalid parameter.</exception>
+        /// <exception cref="OutOfMemoryException">Thrown in case of out of memory.</exception>
+        /// <exception cref="DirectoryNotFoundException">Thrown in case of nonexistence of available resource.</exception>
+        /// <exception cref="InvalidOperationException">Thrown in case of any internal error.</exception>
+        /// <since_tizen> 7.5 </since_tizen>
+        public string GetResControlAllowedResource(string res_type)
```

Comment:
"Res" should be "Resource" instead, as we try to avoid shortened terms in API, but that would repeat the word "Resource", and the summary says it returns some path, but there is nothing in then name about any path,

Reply:
Change function name from "GetResControlAllowedResource"  to "GetResourceControlAllowedResource" and parameter name from "res_type" to "resourceType"